modules/gdscript/gdscript_vm.cpp

```diff
@@ -84,6 +96,21 @@ static String _get_var_type(const Variant *p_var) {
 
 	return basestr;
 }
+
+void GDScriptFunction::_profile_native_call(uint64_t p_t_taken, String p_func_name, String p_instance_class_name) {
```

Comment:
```suggestion
void GDScriptFunction::_profile_native_call(uint64_t p_t_taken, const String &p_func_name, const String &p_instance_class_name) {
```
Missed here when applying suggestions